routes/v1.js

```diff
@@ -384,7 +386,7 @@ api.declare({
 
 /** Define tasks */
 api.declare({
-  method:     'post',
+  method:     'put',
```

Comment:
OK I hadn't realised that the `POST` was to a different location to the `GET`.

I think we can still fix this.

At the moment `createTask` is doing this right, with a `PUT` against `/task/<taskId>`. I propose we backwardly-compatibly modify the `createTask` request payload schema, adding a new property `scheduled` as boolean, default: `true`. Then, we would deprecate `defineTask` API endpoint, until at some point we can remove it altogether. The new mechanism to create a task without scheduling it would be to call `createTask` endpoint, with `"scheduled": false`. Then it is using PUT, respecting the correct resource location, and is truly idempotent and REST-compliant.

What do you think?